content/spin/dynamic-configuration.md

```diff
@@ -166,3 +167,30 @@ url = "redis://localhost"
 ```
 
 You must individually grant each component access to the stores that it needs to use. To do this, use the `component.key_value_stores` entry in the component manifest within `spin.toml`. See [Spin Key Value Store](kv-store-api-guide.md) for more details. 
+
+### SQLite Storage Runtime Configuration
+
+Spin provides built-in SQLite storage. By default, this is backed by a database that Spin creates for you. However, you can use the Spin runtime configuration file (`runtime-config.toml`) to add and customize local SQLite databases.
```

Comment:
```suggestion
Spin provides built-in SQLite storage. By default, this is backed by a database that Spin creates for you. However, you can use the Spin runtime configuration file (`runtime-config.toml`) to point to other SQLite databases.
```

Runtime config will eventually support non-local SQLite databases. We could wait until we actually have that support, but I think it would also be fine to just remove the "local" qualifier.

Reply:
I retained "add and customise" to hint that you can both add new entries and modify the default one, but have removed "local."